projects/core/directives/dropdown/dropdown-position.directive.ts

```diff
@@ -10,27 +9,22 @@ import {TUI_VIEWPORT} from '@taiga-ui/core/tokens';
 import {TuiPoint, TuiVerticalDirection} from '@taiga-ui/core/types';
 
 import {TuiDropdownDirective} from './dropdown.directive';
-import {TUI_DROPDOWN_OPTIONS, TuiDropdownOptions} from './dropdown-options.directive';
+import {TUI_DROPDOWN_OPTIONS} from './dropdown-options.directive';
 
-@Directive({
-    selector: '[tuiDropdown]:not([tuiDropdownCustomPosition]):not([tuiDropdownSided])',
-    providers: [tuiAsPositionAccessor(TuiDropdownPositionDirective)],
-})
+@Directive({standalone: true, selector: '[tuiDropdownPosition]'})
 export class TuiDropdownPositionDirective extends TuiPositionAccessor {
+    private readonly options = inject(TUI_DROPDOWN_OPTIONS);
+    private readonly viewport = inject(TUI_VIEWPORT);
+
     private previous?: TuiVerticalDirection;
 
     readonly type = 'dropdown';
+    readonly accessor = tuiFallbackRectAccessor('dropdown')(
+        inject(TuiRectAccessor) as any as readonly TuiRectAccessor[],
```

Comment:
😢

Reply:
Unfortunately that's the "official" way to handle multi token providers and `inject`